core/domain/classifier_services.py

```diff
@@ -127,6 +127,70 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+
+
+def add_to_training_queue(exploration):
+    """Checks all states of the exploration for re-training conditions and
+    creates ClassifierTrainingJobModel instances in the datastore
+    accordingly.
+
+    Args:
+        exploration: Exploration. Domain object for an exploration.
+    """
+    states = exploration.states
+    for state_name in states:
+        state = states[state_name]
+        if not state.can_undergo_classification():
+            continue
```

Comment:
I have no preference. I'm fine with the current code; it seems readable to me.

EDIT: after writing the above I noticed there was another continue down below.That does start to get a bit iffy... I think it's fine to leave this one in (it avoids an extra level of indentation) but I would change the one below to just a standard if/else.